### PR TITLE
Add `man` page

### PR DESCRIPTION
**Pull request by alexmyczko**

### CODE REVIEW

File: vfit.1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.TH vfit "1" "December 2021" "User Commands"
+.SH NAME
+vfit \- generate backwards-compatible, static instances of variable fonts
+.SH SYNOPSIS
+vfit [\-h] [\-o PATH] [\-f {woff,woff2}] [\-C] config source
+.SH DESCRIPTION
+This (Variable Font Instancing Tool) allows you to generate custom, static
```
Comment (jonpalmisc) [inline vfit.1]: Could you change "This (Variable Font Instancing Tool)" to just "VFIT"?

File: vfit.1 [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+instances of a variable font defined in a configuration file.
+.SH OPTIONS
+.PP
+Generate backwards\-compatible, static instances of variable fonts.
```
Comment (jonpalmisc) [inline vfit.1]: This line should be removed (already stated above and isn't "options"-related)

File: vfit.1 [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+.SS "positional arguments:"
+.TP
+config
+the metadata/style definition file
+.TP
+source
```
Comment (jonpalmisc) [inline vfit.1]: I believe "config" and "source" should be bold (like the flags) so they are more easily distinguishable from their description text?
image

File: vfit.1 [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+show this help message and exit
+.TP
+\fB\-o\fR PATH
+where to place output files
+.TP
+\fB\-f\fR {woff,woff2}
+which format to output as
+.TP
+\fB\-C\fR
+fix contours for macOS (side effects unknown)
```
Comment (jonpalmisc) [inline vfit.1:31]: Could you add an author section with the text "Jon Palmisciano (@jonpalmisc)" as well as a link to this repository so users can find the source code?

**Review by jonpalmisc — CHANGES_REQUESTED** [review]
I've left some comments regarding the addition of the `man` page — please keep in mind that I'm not super familiar with the Debian packaging process nor the conventions for writing `man` pages, so I apologize if some of these requests are ignorant, but there are a few changes I'd like made before merging.

File: vfit.1 [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+vfit \- generate backwards-compatible, static instances of variable fonts
+.SH SYNOPSIS
+vfit [\-h] [\-o PATH] [\-f {woff,woff2}] [\-C] config source
+.SH DESCRIPTION
+This (Variable Font Instancing Tool) allows you to generate custom, static
+instances of a variable font defined in a configuration file.
+.SH OPTIONS
+.PP
+Generate backwards\-compatible, static instances of variable fonts.
+.SS "positional arguments:"
```
Comment (jonpalmisc) [inline vfit.1]: Capitalize first letter? "Positional arguments:"

File: vfit.1 [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+.TP
+config
+the metadata/style definition file
+.TP
+source
+the font to generate instances of
+.SS "optional arguments:"
```
Comment (jonpalmisc) [inline vfit.1]: Capitalize first letter? "Optional arguments:"

File: vfit.1 [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+Generate backwards\-compatible, static instances of variable fonts.
+.SH OPTIONS
+.PP
+.SS "Positional arguments:"
+.TP
+[\config]
+the metadata/style definition file
+.TP
+[\source]
+the font to generate instances of
```
Comment (jonpalmisc) [inline vfit.1]: Looks like something is broken here? I'm not familiar with the `man` page syntax or else I'd fix it myself. Looks like this for me on Debian:
image

Comment (alexmyczko) [reply vfit.1]: i will have to figure and fix (am on debian too)

**Comment by jonpalmisc** [comment]
Looks good — thanks for your work with this! @alexmyczko

**Pull request merged**